src/archiefvernietigingscomponent/sass/components/_destruction-list-detail.scss

```diff
@@ -20,4 +20,8 @@
     padding-left: 1em;
   }
 
+  @include element("review-comment") {
```

Comment:
This is how it looks on my machine:

* left and right padding doesn't fit other elements in the container. It's very distracting
* comment field should be separated from the info above: it can be a bigger margin between them or the comment section can be moved to the second column of the container, to the right of the information about the last review. 

![image](https://user-images.githubusercontent.com/47745906/109509360-196f9b00-7aa1-11eb-82dd-698a99d120f8.png)

Reply:
I don't know why it initially seemed to work, but I have now added the `!important` next to the padding and now they seem to be what they are meant to be! 
![Screenshot from 2021-03-02 09-26-17](https://user-images.githubusercontent.com/19154114/109619930-9c443480-7b39-11eb-9a86-9c8311d9a7e8.png)